src/spec/items.spec.js

```diff
@@ -76,14 +76,60 @@ describe('items', () => {
     res.body[0].name.should.eq('Excalibur Umbra');
   });
   it('should accommodate alternate languages', async () => {
-    const res = await chai.request(server)
+    let res = await chai.request(server)
       .get('/items/search/excalibur%20umbra')
       .set('Accept-Language', 'zh');
     res.should.have.status(200);
     res.body.should.be.an('array');
     res.body.length.should.be.greaterThan(1);
+    res.body[0].i18n.should.not.have.property('it');
+    res.body[0].name.should.eq('Excalibur Umbra');
+    res.body[0].i18n.should.have.property('zh');
+    res.should.have.header('Content-Language', 'zh');
+
+    res = await chai.request(server)
+      .get('/items/search/excalibur%20umbra?language=zh&only=i18n,name');
+    res.should.have.status(200);
+    res.body.should.be.an('array');
+    res.body.length.should.be.greaterThan(1);
+    res.body[0].i18n.should.have.property('zh');
+    res.body[0].i18n.should.not.have.property('it');
```

Comment:
Added a test for the scenario we were seeing where using a different non-english locale after another has been used causes the i18n to be empty